osu.Game/Online/Spectator/SpectatorClient.cs

```diff
@@ -47,7 +47,7 @@ public abstract partial class SpectatorClient : Component, ISpectatorClient
         /// <summary>
         /// Whether the local user is playing.
         /// </summary>
-        protected bool IsPlaying { get; private set; }
+        protected internal bool IsPlaying { get; private set; }
```

Comment:
What's this modifier change for? Nothing new appears to be using this.

Reply:
Leftover change (boring details is: wanted the test to make sure player called `BeginPlaying` before ending it, but realised I could guarantee it by just ensuring player fully loaded).